components/button.json

```diff
@@ -271,6 +271,24 @@
       "#disabled": {
         "text": "@theme-text-primary-disabled"
       }
+    },
+    "layout": {
```

Comment:
Where are the layout buttons? I can't find them in the Button Figma page

Reply:
They are in the realtime page, https://www.figma.com/file/H19CutixoN8SJGO5EXwcc8/Components---MacOS?node-id=3153%3A216, our UX told me that only realtime use this style so Cass move them seperately